zrml/authorized/src/lib.rs

```diff
@@ -143,22 +189,61 @@ mod pallet {
 
         fn on_dispute(
             _: &[MarketDispute<Self::AccountId, Self::BlockNumber>],
-            _: &Self::MarketId,
-            _: &Market<Self::AccountId, Self::BlockNumber, Self::Moment>,
+            market_id: &Self::MarketId,
+            market: &Market<Self::AccountId, Self::BlockNumber, Self::Moment>,
         ) -> DispatchResult {
-            Ok(())
+            if let MarketDisputeMechanism::Authorized = market.dispute_mechanism {
+                if AuthorizedOutcomeReports::<T>::get(market_id).is_some() {
+                    return Err(Error::<T>::AuthorityAlreadyReported.into());
+                }
```

Comment:
Sorry, that I don't get this. The plan was to disallow new disputes, whenever the Authority made its report.

Reply:
My point is: Why is it allowed to add additional disputes _before_ the authority has submitted its report?

Reply:
I see now. Since, the disputors have no idea, when the authority submits its report, this is a good question. 

So, you are in favour of having one dispute at most? 

Say for example we have the opportunity of multiple disputes before the authority has submitted its report. This leads to the situation that disputors can submit the right outcome twice to get more share of the loosers (assumption is that someone submits wrong outcomes). Yeah, I agree, we should only allow one dispute for `authorized`.

The only point allowing multiple disputes is, that disputes are incentivised whenever there was a wrong outcome.

So another question could be: What is the incentive for the first disputor? Possible solution: He could get the `OracleBond` or the [`OutsiderBond`](https://github.com/zeitgeistpm/zeitgeist/issues/868). 

So for this PR I would go with one possible dispute for `authorized`.